lobsterpy/cli.py

```diff
@@ -148,73 +200,36 @@ def get_parser() -> argparse.ArgumentParser:
         help="overwrites already created INCARs an lobsterins with the give name.",
     )
     # TODO: Add some output arguments: options to supply your own basis
-    output_file_group.add_argument(
-        "--userbasis",
-        "--user-basis",
-        default=None,
-        type=_element_basis,
-        nargs="+",
-        help="This setting will rely on a specific basis provided by the user "
-        "(e.g.,  --userbasis Cr.3d.3p.4s N.2s.2p). Default is None.",
-    )
+    # General matplotlib plotting arguments common to all kinds of plots
 
     plotting_parent = argparse.ArgumentParser(add_help=False)
     plotting_group = plotting_parent.add_argument_group("Plotting")
-
-    broadening_group = plotting_group.add_mutually_exclusive_group()
-    broadening_group.add_argument_group("Broadening")
-    broadening_group.add_argument(
-        "--sigma",
+    plotting_group.add_argument(
+        "-fs",
```

Comment:
Can you check elsewhere what they use? 

Also, the f might be confusing now with the files.